test/OpenTelemetry.Tests/Logs/LoggerFactoryAndResourceBuilderTests.cs

```diff
@@ -0,0 +1,82 @@
+// <copyright file="LoggerFactoryAndResourceBuilderTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+#if !NET461
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Exporter;
+using OpenTelemetry.Resources;
+using Xunit;
+
+namespace OpenTelemetry.Logs.Tests
+{
+    public sealed class LoggerFactoryAndResourceBuilderTests
+    {
+        [Fact]
+        public void VerifyResourceBuilder_DefaultCase()
+        {
```

Comment:
```suggestion
        public void TestLogExporterCanAccessResource()
        {
```

Reply:
with the current helper method, the suggested name would not be accurate.

Reply:
That makes sense. Thanks :)